.circleci/config.yml

```diff
@@ -102,7 +102,7 @@ jobs:
     docker:
       - image: circleci/node:lts
       - image: trufflesuite/ganache-cli
-        command: ganache-cli -i 1234 -l 6720000
+        command: ganache-cli -i 1234 -l 9000000
```

Comment:
Need to increase gas limit to accommodate increased bytecode size thanks to adding ReentrancyGuard.sol